src/main/java/persistence/sql/ddl/TableClause.java

```diff
@@ -0,0 +1,90 @@
+package persistence.sql.ddl;
+
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Stream;
+
+import persistence.entity.exception.InvalidPrimaryKeyException;
+import persistence.sql.ddl.column.ColumnClauses;
+import persistence.sql.exception.NotIdException;
+
+public class TableClause {
+    private final String name;
+    private final PrimaryKeyClause primaryKeyClause;
+    private final ColumnClauses columnClauses;
+    private final Object instanceOfTable;
+
+    public TableClause(Class<?> entity) {
+        this.name = getTableName(entity);
+        this.primaryKeyClause = extractIdFrom(entity);
+        this.columnClauses = extractColumnsFrom(entity);
+        this.instanceOfTable = getInstanceOfTable(entity);
+    }
+
+    private Object getInstanceOfTable(Class<?> entity) {
+        try {
+            return Arrays.stream(entity.getDeclaredConstructors())
+                    .filter(x -> x.getParameterCount() == 0)
+                    .findFirst().get().newInstance();
+        } catch (InvocationTargetException | InstantiationException | IllegalAccessException e){
+            throw new RuntimeException("새로운 인스턴스 생성에 실패하였습니다.");
+        }
+    }
+
+
+    private static ColumnClauses extractColumnsFrom(Class<?> entity) {
+        List<Field> fields = Arrays.stream(entity.getDeclaredFields())
+                .filter(x -> !x.isAnnotationPresent(jakarta.persistence.Id.class))
+                .toList();
+
+        return new ColumnClauses(fields);
```

Comment:
```suggestion
        return new ColumnClauses(의도를드러내는이름함수(fields));
```
복잡한 함수를 의도를 표현하는 이름으로 분리하면 어떨까요?